index.html

```diff
@@ -1,15 +1,28 @@
 <!DOCTYPE html>
-<html>
+<html data-ng-app = "wdiradio">
   <head>
     <title>WDI Radio</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
     <script src="js/app.js"></script>
+    <script src="js/songs/songs.module.js"></script>
+    <script src="js/songs/index.controller.js"></script>
+    <script src="js/songs/show.controller.js"></script>
+    <script src="js/songs/songs.factory.js"></script>
+    <script src="js/songs/edit.controller.js"></script>
+    <script src="js/songs/form.directive.js"></script>
+    <script src="js/songs/new.controller.js"></script>
+    <link rel="stylesheet" href="js/style.css">
 
   </head>
   <body>
-    <header>WDI Radio</header>
+    <h1><a data-ui-sref="songIndex">WDI Radio</a></h1>
```

Comment:
Is this working?  This page is rendered by rails and these angular directives are outside the the ui-view.